P/Polymers/build_tarballs.jl

```diff
@@ -0,0 +1,32 @@
+using BinaryBuilder, Pkg
+
+name = "Polymers"
+version = v"0.3.6"
+sources = [
+    GitSource(
+        "https://github.com/sandialabs/Polymers.git",
+        "c977b84d921019b3731b2c6cce2537e1d03ebcef",
+    ),
+]
+script = raw"""
+cd $WORKSPACE/srcdir/Polymers
+cargo build --release --features extern
+install -Dvm 755 "target/${rust_target}/release/libpolymers.so" "${libdir}/libpolymers.so"
+"""
+platforms = [supported_platforms()[2]]
+products = [LibraryProduct("libpolymers", :libpolymers)]
+dependencies = [Dependency("DocStringExtensions")]
```

Comment:
Why?  This isn't a JLL, I don't see why it should be a dependency
```suggestion
dependencies = Dependency[]

```